src/sidebar/FeedList/Folder.tsx

```diff
@@ -115,47 +154,62 @@ const Folder = (props: Props) => {
             return;
         }
 
-        setDraggedOver(false);
+        if (relativeDropPosition === undefined) {
+            throw new Error('Illegal state: relativeDropPosition must be definend when handleDrop is called.');
```

Comment:
This can currently happen due to timing issues. This method should not rely on the state set be handleDragover because it is only set dependent on props.disabled (which depends on the global store). The local state `relativeDropPosition` is not always correct if drag/drop happens fast. It should only be used for UI indication.

Instead recalculate relativeDropPosition iside handelDrop

Reply:
[v](https://github.com/dermeck/feeds-sidebar/pull/157/commits/b8e5f5b95ea2a61c56373a4ff533f903e54f7a35)